ocaml/testsuite/tests/typing-layouts-float64/basics_alpha.ml

```diff
@@ -405,7 +405,7 @@ Error: This expression has type t_float64
        The layout of t_float64 is float64, because
          of the annotation on the declaration of the type t_float64.
        But the layout of t_float64 must be a sublayout of value, because
-         it's used as a function result.
+         it's used as a function result (defaulted to layout value).
```

Comment:
I guess the defaulting information is an improvement over the status quo, but this still should probably mention the function `id_value` somewhere.